### DISCUSSION TITLE
Should we add optional "effort" labels?

### DISCUSSION THREAD

**Discussion by sarayourfriend** [WordPress/openverse#2816] [Polls]
Replacing https://github.com/WordPress/openverse/issues/296 with a concrete poll.

The consensus there _appears_ to be yes, but I want to verify. If we agree, then we can create an issue to actually make the new labels along with documentation for how to use them.

To refine the proposal, we would be doing the following:

1. Add new effort labels for low and medium effort. Anything unlabelled can be assumed to be high _or sufficiently ambiguous/unplanned such that any effort label wouldn't be helpful_ or that the work has not been evaluated closely enough to judge the effort, regardless of whether it would eventually be labelled low/medium or not (i.e., the labels are optional and there is no obligation or expectation to add them to any issue).
2. Update existing documentation about 'good first issue' and 'help wanted', including maintainer guidance, to describe the above. Ensure that the effort is related to the technology labels.

Additional practices, like whether we would start picking some number of low-effort tickets to put into the todo column each cycle, etc, all warrant their own discussion and do not have a direct bearing on whether the labels are useful generally (e.g., for new contributors).

**Comment by krysal** [comment]
Would this new set of labels be mandatory? I'm not voting "No" because I don't want to block the addition if other folks wish the label, but I'm not voting "Yes" either because I don't think it should be mandatory.

**Reply by sarayourfriend** [reply]
The proposal is for them to be optional, as the title says :slightly_smiling_face: 

I was the only one who voted anyway so I've updated the question to use the word "optional" as well and clarified the unlabelled case in the first point.

**Comment by dhruvkb** [comment]
I don't have a strong opinion on this except that effort/difficulty etc. are subjective and can change with familiarity with a particular language, tool or framework. What is low effort or easy for someone might be high effort or difficult for someone else, especially if it needs the contributor to read docs and learn something new before even starting to work on something.

For example, for a new contributor with background in Vue, a Nuxt issue might be trivial, but for someone with say a React background, the learning curve can be steep. Seeing an "effort: low" label in such a scenario can be quite discouraging.

**Reply by sarayourfriend** [reply]
That's very true. It would be really discouraging to see that if I was new to something. More so even than "good first issue", I think.

**Comment by sarayourfriend** [comment]
Closing this. I think we basically landed on not adding them for the reason Dhruv shared. I would change my vote to "no", in any case.
